config/locales/referee_interface/referee_interface.yml

```diff
@@ -2,7 +2,7 @@ en:
   referee:
     refuse_feedback:
       choice:
-        label: Can you give %{full_name} a reference?
+        label: Can you give a reference for %{full_name}?
         confirm: No, I decline to give a reference
```

Comment:
I've just noticed on the prototype this is `No, I am unable to give a reference`:
![image](https://user-images.githubusercontent.com/47917431/142405838-3c98ad4f-44a3-4181-87da-ec75983bf1ad.png)